buildpipeline/pipeline.json

```diff
@@ -85,7 +85,7 @@
             "PB_DockerTag": "rhel7_prereqs_2",
             "PB_BuildArguments": "-buildArch=x64 -Release -portableLinux",
             "PB_SyncArguments": "-p -portableLinux -- /p:ArchGroup=x64",
-            "PB_TargetQueue": "Redhat.72.Amd64+Debian.82.Amd64"
+            "PB_TargetQueue": "Redhat.72.Amd64+Debian.82.Amd64+Ubuntu1404.Amd64+Ubuntu1604.Amd64+Ubuntu.1610.Amd64+suse.421.amd64"
```

Comment:
@gkhanna79 @chulbert is still working on Fedora25.  It was having issues but is likely ready by the end of the week.  Should I enable it for 23/24 in the meantime?  That wasn't part of the targets in the notes.